assets/js/admin-permissions.js

```
@@ -0,0 +1,841 @@
+                    object_type: objectType,
+                    object_id: objectId,
+                    relation: relation
+                })
+            });
+
+            if (!response.ok) {
+                let errorMsg = config.i18n.failedToGrant;
+                try {
+                    const data = await response.json();
+                    errorMsg = data.message || data.error || errorMsg;
+                } catch (e) { /* non-JSON response */ }
```

ESLint

[error] 'e' is defined but never used.

(no-unused-vars)